src/Policy/OrmResolver.php

```diff
@@ -69,7 +69,7 @@ public function getPolicy($resource)
             return $this->getRepositoryPolicy($resource);
         }
         if ($resource instanceof QueryInterface) {
-            return $this->getRepositoryPolicy($resource->repository());
+            return $this->getRepositoryPolicy($resource->getRepository());
```

Comment:
`getRepository()` does not exist in cake3.5 
first you should check for availability

Reply:
Instead of the antipattern of method joggling/testing, I would rather raise the limit to 3.6+ at this point then.